### PR TITLE
mentioned the mergeParams in the routing file.

### PR DESCRIPTION
**Pull request by Ahmed1monm**

### CODE REVIEW

**Review by IamLizu — CHANGES_REQUESTED** [review]
Hey @Ahmed1monm 👋 

Thank you for your contribution. It looks like there's a small typo in the starting sentence. 

Could you please re-check and see if removing "by" from "But if the parent..." makes it more correct?

**Comment by Ahmed1monm** [comment]
@IamLizu Thank you for your comment. I fixed it

**Review by IamLizu — APPROVED** [review]

**Comment by Ahmed1monm** [comment]
Good afternoon @IamLizu thank you for the approval 😃.  May I ask if the PR will be merged and when?

File: en/guide/routing.md [hunk 1]
```diff
@@ -354,3 +354,10 @@ app.use('/birds', birds)
 ```
 
 The app will now be able to handle requests to `/birds` and `/birds/about`, as well as call the `timeLog` middleware function that is specific to the route.
+
+But if the parent rout `/birds` has path params, it will not be accessible by default from the sub-routes,
```
Comment (crandmck) [inline en/guide/routing.md]: Spelling: "rout" -> "route"

Please spell out "parameters".

Comma-splice:
"by default from the sub-routes, to make it..."

s/be: "by default from the sub-routes. To make it..."

**Review by crandmck — CHANGES_REQUESTED** [review]

File: en/guide/routing.md (hunk 1 above)
Comment (Ahmed1monm) [reply en/guide/routing.md]: Thank you for the review, I fixed it.

**Review by crandmck — APPROVED** [review]

**Comment by ahmedesmail07** [comment]
what a great contribution!!

**Pull request merged**